devbot/bot.py

```diff
@@ -68,24 +71,28 @@ def main():
 
 
 def logging_setup():
+    file_level = logging.DEBUG
+    console_level = logging.INFO
+    if "FILE_LOGLEVEL" in os.environ.keys():
+        file_level = os.environ["FILE_LOGLEVEL"]
+    if "CONSOLE_LOGLEVEL" in os.environ.keys():
+        console_level = os.environ["CONSOLE_LOGLEVEL"]
     # Set up basic functions to log to a file
-    logging.basicConfig(level=logging.DEBUG,
+    logging.basicConfig(level=file_level,
```

Comment:
This doesn't seem right. Does `logging.basicConfig()` accept `level=logging.DEBUG` and 'level="DEBUG" interchangeably?

Reply:
Yes,over tested it and it works